ChangeLog

```diff
@@ -11,6 +11,11 @@ Release date: TBA
 ..
   Put new features here and also in 'doc/whatsnew/2.12.rst'
 
+* Fix ``install graphiz`` message which isn't needed for puml output format.
+
+  Closes #
+
+
```

Comment:
If no ticket exists the ``Closes #`` is not necessary. Also only a single empty line between two ChangeLog entries is the "standard".
As a heads up: it's better if you apply those changes locally on your machine, as the GitHub web UI is not reliable when it comes to adding/removing blank lines. So trying to apply my suggestion directly here on GitHub will probably still be faulty at the end. 

```suggestion
* Fix ``install graphiz`` message which isn't needed for puml output format.

```